modules/core/src/main/scala/ch/epfl/bluebrain/nexus/iam/core/acls/Permissions.scala

```diff
@@ -92,10 +92,18 @@ object Permissions {
   /**
     * Convenience factory method to construct a [[ch.epfl.bluebrain.nexus.iam.core.acls.Permissions]] instance from var args.
     *
-    * @param items the permissions included in the resulting set
-    * @return a new __Permissions__ instance that contains all argument __permissions__
+    * @param items a non-empty permissions set to be included in the resulting instance
+    * @return a new __Permissions__ instance
     */
-  def apply(items: Permission*): Permissions = apply(items.toSet)
+  def apply(items: Permission*): Permissions = {
+    require(items.nonEmpty, "Argument 'items' must not be empty, use Permissions.empty explicitly instead")
```

Comment:
If we conceptually allow permissions to be empty, this require seems wrong...

Reply:
Right, I wanted to avoid creating empty Permissions by mistake but this doesn't help much.